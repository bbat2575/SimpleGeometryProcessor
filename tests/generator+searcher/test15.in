-rseed=1 -mindist=2 -N=20