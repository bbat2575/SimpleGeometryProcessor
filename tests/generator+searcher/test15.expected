read 20 points
33.58, -6.72
43.91, -11.88
33.76, 5.65
This is a triangle
